lib/bn.js

```diff
@@ -1021,6 +1237,12 @@ BN.prototype.mul = function mul(num) {
   return this.mulTo(num, out);
 };
 
+// Multiply employing FFT
+BN.prototype.mulf = function mulf(num) {
+  var out = new BN(null);
+  return jumboMulTo(this, num, out);
```

Comment:
No, i may not, it hugely depends on the length of the chunk that the source `words` are broken into, that's why there're method `guessLen13b`.

And i still don't get why we need this though?